ebu_tt_live/documents/converters.py

```diff
@@ -41,7 +42,9 @@ def ebutt1_to_ebutt3(ebutt1_in):
     :return:
     """
     converter = EBUTT1EBUTT3Converter()
-    ebutt3_bindings = converter.convert_document(ebutt1_in.binding)
+    doc_xml = ebutt1_in.get_xml()
+    ebutt1_doc = EBUTT1Document.create_from_xml(doc_xml)
+    ebutt3_bindings = converter.convert_document(ebutt1_doc.binding)
```

Comment:
This bit confuses me. Here we're taking an EBUTT1Document object, getting its XML, reading that back into an EBUTT1Document object, then converting it to EBUTT3.
The XML-and-back-again conversion seems unnecessary. Any particular reason for this?

Reply:
It's consistent with the `ebutt3_to_ebuttd` function, but I agree it isn't clear why this approach is taken in either place. Also it looks to me as though neither function is ever even called.